pkg/feature/conditions.go

```diff
@@ -102,6 +104,18 @@ func WaitForPodsToBeReady(namespace string) Action {
 	}
 }
 
+func filterEvictedPods(pods []corev1.Pod) []corev1.Pod {
+	var filteredPods []corev1.Pod
+
+	for _, pod := range pods {
+		if pod.Status.Phase != corev1.PodFailed || pod.Status.Reason != "Evicted" {
```

Comment:
isnt `if pod.Status.Phase != corev1.PodFailed  {` enough
or your evicted pod was in non-Failed phase?

Reply:
the pod was in Failed state, but do we want to filter out all pods in failed state? it might be that some of these are an after-effect of the apply we do from our side, as opposed to the ones that failed because of eviction

Reply:
ok, so basically the logic is to filter out 
pod is in `Failed` status  with a reason of `Evicted`
`if !(pod.Status.Phase == corev1.PodFailed && pod.Status.Reason == "Evicted") { // TO KEEP ...}`

Reply:
yes, exactly